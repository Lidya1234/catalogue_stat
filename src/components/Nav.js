const Nav = () =>
{
    return(
        <ul>
            <li>A</li>
            <li>B</li>
        </ul>
    )
}
export default Nav;