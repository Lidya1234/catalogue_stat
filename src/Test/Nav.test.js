import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../components/Nav';
import '@testing-library/jest-dom/extend-expect';

describe('The navigation bar should contain', () => {
  it('Meal-Categories logo', () => {
    const { getByText } = render(
      <MemoryRouter>
        <Nav />
      </MemoryRouter>,
    );
    const link = getByText('Meal-Categories');
    expect(link).toBeInTheDocument();
  });
  const component = renderer.create(
    <MemoryRouter>
      <Nav />
    </MemoryRouter>,
  ).toJSON();

  test('Matchs snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
