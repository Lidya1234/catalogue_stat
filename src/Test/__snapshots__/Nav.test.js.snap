// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The navigation bar should contain Matchs snapshot 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <a
    className="navbar-brand"
    href="#home"
  >
    Meal-Categories
  </a>
</nav>
`;
